qubes/api/admin.py

```diff
@@ -899,9 +899,9 @@ def vm_start(self):
         scope='local', execute=True)
     @asyncio.coroutine
     def vm_shutdown(self):
-        self.enforce(not self.arg)
         self.fire_event_for_permission()
```

Comment:
`fire_event_for_permission()` should also get `force` parameter, so extensions can make more informed decision.